content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -1214,12 +1216,12 @@ When `chat.users` is defined, one of following scopes is required:
 
 </CodeSample>
 
-<!-- > **`activate_chat`** sample **request** with optional params
+<!-- > **`resume_chat`** sample **request** with optional params
```

Comment:
This is some legacy stuff that doesn't need to be updated. Feel free to delete it next time you come across it.